bucket/cura.json

```
@@ -2,36 +2,32 @@
     "version": "4.2.1",
     "license": "LGPL-3.0-only",
     "homepage": "https://ultimaker.com/en/products/ultimaker-cura-software",
-    "description": "Ultimaker Cura prepares your model for 3D printing.",
-    "bin": "cura.exe",
+    "description": "Model editing tools for 3D printing",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/Ultimaker/Cura/releases/download/4.2.1/Ultimaker_Cura-4.2.1-win64.exe",
+            "url": "https://github.com/Ultimaker/Cura/releases/download/4.2.1/Ultimaker_Cura-4.2.1-win64.exe#/dl.7z",
             "hash": "b463873df49434f2dbc960746a92a0767fe0477a80bf86903624f4b51cf4e711"
         }
     },
+    "suggest": "extras/vcredist2015",
```

Suggest is not string

```suggestion
    "suggest": {
            "Microsoft Visual C++ Redistributables 2015": "extras/vcredist2015"
        }
```